gobblin-runtime/src/main/java/org/apache/gobblin/runtime/template/StaticJobTemplate.java

```diff
@@ -64,13 +64,23 @@
   private Collection<String> dependencies;
 
   public StaticJobTemplate(URI uri, String version, String description, Config config, JobCatalogWithTemplates catalog)
-      throws SpecNotFoundException, TemplateException {
+      throws TemplateException {
     this(uri, version, description, config, getSuperTemplateUris(config), catalog);
   }
 
+  /** An constructor that materialize multiple templates into a single static template
+   * The constructor provided multiple existing templates as the input instead of templateURIs
```

Comment:
`multiple in-memory templates` instead of `multiple existing templates`?

Reply:
Addressed.